common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -53,7 +53,9 @@ function (VideoPlayer, VideoStorage, i18n) {
                         }
 
                         _hideWaitPlaceholder(state);
-                        state.el.trigger('initialize', arguments);
+                        var vent = document.createEvent('Event');
```

Comment:
I'm not sure `createEvent` is compatible with Internet Explorer.
see: https://developer.mozilla.org/en-US/docs/Web/API/Document/createEvent

we need to test this manually, as our automation doesn't run against IE.